draft/draft-scharf-tcpm-yang-tcp.xml

```diff
@@ -174,66 +195,44 @@
         define a given configuration parameter globally, while another one
         uses per-interface settings, and both approaches work well for the
         corresponding use cases. Also, different systems may use different
-        default values.</t>
+        default values. In addition, TCP can be implemented in different ways
+        and design choices by the protocol engine often affect configuration
+        options.</t>
 
-        <t>In addition, TCP can be implemented in different ways and design
-        choices by the protocol engine often affect configuration options. In
-        a number of areas there are known differences between different TCP
-        stack architectures. This includes amongst others:</t>
+        <t>Nonetheless, a number of TCP stack parameters require configuration
+        by YANG models. This document therefore defines a minimal YANG model
+        with fundamental parameters directly following from TCP standards.</t>
 
-        <t><list style="symbols">
-            <t>Window size: TCP stacks can either store window state variables
-            (such as the congestion window) in segments or in bytes.</t>
-
-            <t>Buffer sizes: The memory management depends on the operating
-            system. As the size of buffers can vary over several orders of
-            magnitude, very different implementations exist. This typically
-            influences TCP flow control.</t>
-
-            <t>Timers: Timer implementation is another area in which TCP
-            stacks may differ.</t>
-          </list></t>
-
-        <t>Nonetheless, there are a number of basic system parameters that are
-        configurable on many TCP implementations, even if not all TCP
-        implementations may indeed have all these settings, and even if there
-        are differences regarding syntax and semantics. This document focuses
-        on modeling such basic parameters directly following from
-        standards.</t>
+        <t>An important use case is the TCP configuration on network elements
+        such as routers, which often use YANG data models. The model therefore
+        specifies TCP parameters that are important on such TCP stacks. A
+        typical example is the support of <xref
+        target="RFC5925">TCP-AO</xref>. TCP-AO is increasingly supported on
+        routers and it requires configuration.</t>
```

Comment:
You could add that "TCP-AO is increasingly supported on routers to secure routing protocols such as BGP, and it requires that the routing protocol configure the router for that purpose."